modules/dreamview/frontend/src/components/StatusBar/Electricity.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { observer } from 'mobx-react';
+
+@observer
+export default class Electricity extends React.Component {
+
+  render() {
+    const {
+      electricityPercentage,
+    } = this.props;
+
+    const percentageString = `${electricityPercentage}%`;
+    const rectWidth = 24 * (electricityPercentage / 100);
+    return (
+    <div className="show-electricity">
+        <div className="battery-icon">
+	    <svg version="1.1">
+              <rect rx="2" ry="2" x="50" y="10" width="30" height="15" fill="rgb(15,127,18)">
```

Comment:
i think you don't need the close tag "\</rect>". Just add "/>" to the end of the open tag.

Reply:
got it, thank u.